### PR TITLE
9th step -  Remove outputFilter and outputKey

### PR DESCRIPTION
**Pull request by krhubert**

### CODE REVIEW

File: commands/provider/wallet_provider.go [hunk 1]
```diff
@@ -96,9 +96,9 @@ func (p *WalletProvider) call(task string, inputs interface{}, output interface{
 }
 
 func (p *WalletProvider) parseResult(r *coreapi.ResultData, output interface{}) error {
-	if r.OutputKey == "error" {
+	if r.Error != "error" {
```
Comment (NicolasMahe) [inline commands/provider/wallet_provider.go]:
```suggestion
	if r.Error != "" {
```
missing modif here

Comment (NicolasMahe) [reply commands/provider/wallet_provider.go]: @krhubert why this comment is solved?

Comment (krhubert) [reply commands/provider/wallet_provider.go]: Because i forced pusehd when I changed it

Comment (krhubert) [reply commands/provider/wallet_provider.go]: I will fix it in 11th step. so now we won't trigger again tests and builds for 2 branches

**Pull request merged**